src/reset.css

```diff
@@ -0,0 +1,10 @@
+body,
+div,
+img,
+h1,
+h2,
+p,
+a {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
You can use box-sizing for `*` selector to apply this rule for all elements.
But your solution is ok here.